client/src/app/pages/archetypes/archetypes-page.tsx

```diff
@@ -115,34 +120,37 @@ const Archetypes: React.FC = () => {
     onError
   );
 
-  const { mutate: deleteAssessment } = useDeleteAssessmentMutation();
+  const { mutateAsync: deleteAssessment } = useDeleteAssessmentMutation();
 
   const discardAssessment = async (archetype: Archetype) => {
-    try {
-      if (archetype.assessments) {
-        await Promise.all(
-          archetype.assessments.map(async (assessment) => {
-            await deleteAssessment({
-              assessmentId: assessment.id,
-              archetypeId: archetype.id,
-            });
-          })
-        ).then(() => {
-          pushNotification({
-            title: t("toastr.success.assessmentDiscarded", {
-              application: archetype.name,
-            }),
-            variant: "success",
-          });
+    if (!archetype.assessments) {
+      return;
+    }
+    Promise.all(
+      archetype.assessments.map(async (assessment) => {
+        await deleteAssessment({
+          assessmentId: assessment.id,
+          archetypeId: archetype.id,
+        });
+      })
+    )
+      .then(() => {
```

Comment:
Nit-pick: With the promise callback style, the function doesn't need to be `async`.